estate/models/estate_property.py

```diff
@@ -0,0 +1,151 @@
+from datetime import datetime, timedelta
+from odoo import fields, models, Command
+from odoo import api
+from odoo import exceptions
+from odoo.exceptions import ValidationError
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "ESTATE Properties"
+    _order = "id desc"
+
+    name = fields.Char(string="Title", required=True, help="Property name")
+    description = fields.Text(string="Description", help="Property description")
+    postcode = fields.Text(string="Postcode", help="Property Post code")
```

Comment:
```suggestion
    postcode = fields.Text(help="Property Post code")
```
por defecto si no pones un argumento `string`, odoo va a poner como string el nombre del campo, poniendole mayusculas al inicio de las palabras y reemplazando los `_` con espacios